README.md

```diff
@@ -50,8 +50,9 @@ http_archive(
     sha256 = "8c48283aeb70e7165af48191b0e39b7434b0368718709d1bced5c3781787d8e7",
 )
 
-load("@io_bazel_rules_scala//:version.bzl", "bazel_version")
-bazel_version(name = "bazel_version")
+# 
```

Comment:
Leftover?

Reply:
Actually a mistake - I forgot to add documentation text here